app/views/alchemy/pages/_meta_data.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @page %>
+  <meta charset="utf8">
+  <%= content_tag :title, page_title(
```

Comment:
Without the `content_tag` helper this will actually be better readable and the helper does not really provide any value here. `<title>` is better here